manifests/base/tasks/run.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
How is this task different from #10 and why is it part of this PR (since #10 is about tekton, not this one?)

Reply:
Accidentally added file to the wrong branch.